Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx

```diff
@@ -0,0 +1,55 @@
+import AlertBox from "../AlertBox/AlertBox";
+import SectionText from "../SectionText/SectionText";
+import SectionTitle from "../SectionTitle/SectionTitle";
+import "./ContentCenter.css";
+
+const ContentCenter = () => {
+  return (
+    <div style={{ display: "flex", justifyContent:"space-between" }} className="content-center">
```

Comment:
@jhonmartinezG , using inline styles is a bad practice please move this to class, also this div should be a section